src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -195,24 +195,26 @@
 Connected_Cluster_Resource = "connected_cluster_resource_snapshot.txt"
 DNS_Check = "dns_check.txt"
 K8s_Cluster_Info = "k8s_cluster_info.txt"
-Outbound_Network_Connectivity_Check = "outbound_network_connectivity_check.txt"
+Outbound_Network_Connectivity_Check_for_onboarding = "outbound_network_connectivity_check_for_onboarding.txt"
+Outbound_Network_Connectivity_Check_for_cluster_connect = "outbound_network_connectivity_check_for_cluster_connect.txt"
 Events_of_Incomplete_Diagnoser_Job = "diagnoser_failure_events.txt"
 # Connect Precheck Diagnoser constants
-Cluster_Diagnostic_Checks_Job_Registry_Path = "mcr.microsoft.com/azurearck8s/helmchart/stable/clusterdiagnosticchecks:0.1.1"
+Cluster_Diagnostic_Checks_Job_Registry_Path = "connectprecheck.azurecr.io/helm/new-pre-onboarding-inspector:0.1.13-new"
 Cluster_Diagnostic_Checks_Helm_Install_Failed_Fault_Type = "Error while installing cluster diagnostic checks helm release"
 Cluster_Diagnostic_Checks_Execution_Failed_Fault_Type = "Error occured while executing cluster diagnostic checks"
 Cluster_Diagnostic_Checks_Release_Cleanup_Failed = "Error occured while cleaning up the cluster diagnostic checks helm release"
 Cluster_Diagnostic_Checks_Job_Not_Scheduled = 'Unable to schedule cluster-diagnostic-checks job'
 Cluster_Diagnostic_Checks_Job_Not_Complete = 'Unable to complete cluster-diagnostic-checks job after scheduling'
 Pre_Onboarding_Diagnostic_Checks_Execution_Failed = 'Exception occured while trying to execute pre-onboarding diagnostic checks'
-Outbound_Connectivity_Check_Failed = "Outbound network connectivity check failed"
+Outbound_Connectivity_Check_Failed_For_Onboarding = "Outbound network connectivity check for onboarding failed"
 DNS_Check_Failed = "DNS Resolution failed"
 Cluster_Diagnostic_Prechecks_Failed = "Cluster diagnostic prechecks failed"
 Cluster_Diagnostic_Prechecks_Incomplete = "Cluster diagnostic prechecks failed to complete"
 Cluster_Diagnostic_Checks_Pod_Description_Save_Failed = 'Failed to save cluster diagnostic checks pod description'
 Cluster_Diagnostic_Checks_Job_Log_Save_Failed = 'Failed to save cluster diagnostic checks job log'
 # Diagnostic Results Name
-Outbound_Connectivity_Check_Result_String = "Outbound Network Connectivity Result:"
+Outbound_Connectivity_Check_Check_String = "Outbound Network Connectivity Check"
+Outbound_Connectivity_Check_Failed_For_Cluster_Connect_Endpoint = "Outbound network connectivity check failed for Cluster Connect endpoint"
```

Comment:
let's remove the "endpoint" in the end since it might be confusing. It is just one of the endpoints needed for cluster connect feature

Reply:
Also, nit: can we keep 209 and 217 consistent? the word "failed" is a two different positions